### ISSUE TITLE
How to make isometric camera ?

### ISSUE THREAD

**Issue by SkybuckFlying** [mmp/pbrt-v4#261]
It would involve rotating the matrix:

Rotate 35.264 1.0 0.0 0.0
Rotate -45.0  0.0 1.0 0.0

I tried with a lookat setup:

# eye
# look at point
LookAt 
	0 25 0
	0 0 0
	0 0 1
# up vector

But this doesn't look right... the z-axis is pointing right.

The y-axis is now pointing to the sky.

It looks cool, but does not resembly the open-x-com game.

(I want z-axis to point upwards into the sky, and the x and y to be on the ground...)

**Comment by SkybuckFlying** [comment]
So far I was using perspective camera, but using ortho might lead to something, now I need a way to "zoom out"

Camera "orthographic"	


Experimenting with translate before and after rotate, not yet sure which order to use.

**Comment by shadeops** [comment]
If you want to increase the size of your orthographic camera so it captures more of the scene you need to increase the camera's screenwindow parameter.

If the resolution of your image plane is 512x512, and you want to capture the scene 10 units around your orthgraphic project then you would set 
```
Camera "orthographic" "float screenwindow" [ -5 5 -5 5 ]
```


ie)
```
Film "rgb" "string filename" [ "ortho_2.png" ] "integer xresolution" [ 320 ] "integer yresolution" [ 240 ]
Camera "orthographic" "float screenwindow" [ -1 1 -0.75 0.75 ]
```
![ortho_2](https://user-images.githubusercontent.com/2104033/169421270-453c98f0-b604-435c-9020-05622818fa14.png)


```
Film "rgb" "string filename" [ "ortho_5.png" ] "integer xresolution" [ 320 ] "integer yresolution" [ 240 ]
Camera "orthographic" "float screenwindow" [ -2.5 2.5 -1.875 1.875 ]
```
![ortho_5](https://user-images.githubusercontent.com/2104033/169421355-c3dba509-07c5-4f7b-bfad-99e243ce94af.png)

**Comment by SkybuckFlying** [comment]
Thank you very much for helping me, with your help, I think I have achieved in recreating something that looks similiar to the OpenXcom look, by using these rotations and your suggested camera window fix. (I think the look at is also necessary)

LookAt 
	0 25 0
	0 0 0
	0 0 1

Camera "orthographic"	
	"float screenwindow" [ -5 5 -5 5 ]

Rotate -35.264 1.0 0.0 0.0
Rotate 45 0.0 0.0 1.0

Notice the last rotation is a bit different then the original one posted. Fixed that one myself, miraculously.
